src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/reactTable/components/cells/CellPublicationStatus.jsx

```diff
@@ -3,6 +3,8 @@ import {TableBodyCell} from '@jahia/moonstone';
 import PublicationStatus from '../../../../PublicationStatus';
 import PropTypes from 'prop-types';
 import {withStyles} from '@material-ui/core';
+import scssStyles from './Cells.scss';
```

Comment:
Yeah I considered that as well, but this one is a bit complicated. Since the component is also using `withStyles` from Material UI which also injects a `classes` variable and probably cause a collision. Ideally probably better to get rid of the material UI withStyles altogether and stick with the scss but chose not to do the refactor coz it's a bit out of scope for the story. But I'll add a note to the ticket for future consideration.